### DISCUSSION TITLE
How to define a HasOne relationship?

### DISCUSSION THREAD

**Discussion by it-dubbelglas** [filamentphp/filament#1761] [Help] [answered]
Hi,

First of all; I've just discovered this package and I think it is amazing!

While setting up my project I came accross the following situation:

I have a Customer and Company model which define a HasOne relationship, a customer can have one company. Though, nowhere in the documentation I could find how to set that up.

Is it possible to have a HasOne-relationship(-manager) ?

**Comment by danharrin** [comment, marked as answer]
#2304
